.github/workflows/linux.yml

```diff
@@ -307,15 +307,15 @@ jobs:
     uses: ./.github/workflows/job_debian_packages.yml
     with:
       runner: 'aks-linux-4-cores-16gb'
-      image: 'openvinogithubactions.azurecr.io/dockerhub/ubuntu:20.04'
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_22_04_x64 }}
```

Comment:
I suppose we need to keep _empty_ image to check OpenVINO installation on empty platform.

Reply:
Makes sense, returned.